### ISSUE TITLE
Bug in argh/dispatching.py?

### ISSUE THREAD

**Issue by cdunn2001** [bioconda/bioconda-utils#323]
Posting in case this is a bug. (I don't actually need the `dependent`  command. I'm only trying to figure out bioconda/bioconda-recipes#10264 )

    $ bioconda-utils dependent recipes/ config.yml

```py
Traceback (most recent call last):
  File "/tmp/miniconda/miniconda/bin/bioconda-utils", line 11, in <module>
    sys.exit(main())
  File "/tmp/miniconda/miniconda/lib/python3.6/site-packages/bioconda_utils/cli.py", line 638, in main
    bioconductor_skeleton, pypi_check, clean_cran_skeleton,
  File "/tmp/miniconda/miniconda/lib/python3.6/site-packages/argh/dispatching.py", line 328, in dispatch_commands
    dispatch(parser, *args, **kwargs)
  File "/tmp/miniconda/miniconda/lib/python3.6/site-packages/argh/dispatching.py", line 174, in dispatch
    for line in lines:
  File "/tmp/miniconda/miniconda/lib/python3.6/site-packages/argh/dispatching.py", line 277, in _execute_command
    for line in result:
  File "/tmp/miniconda/miniconda/lib/python3.6/site-packages/argh/dispatching.py", line 260, in _call
    result = function(*positional, **keywords)
  File "/tmp/miniconda/miniconda/lib/python3.6/site-packages/bioconda_utils/cli.py", line 502, in dependent
    for pkg in packages:
UnboundLocalError: local variable 'packages' referenced before assignment
```
I *think* I have an up-to-date minoconda, via bioconda-recipes/bootstrap.

**Comment by dpryan79** [comment]
There are a couple bugs here. The first one you've run into, which is that `packages` should be `d`. The second is that one must specify one of `--reverse-dependencies` or `--dependencies`, which isn't indicated in the usage.

**Comment by epruesse** [comment]
Fixed in #410
